docker-compose-metrics.yml

```diff
@@ -0,0 +1,23 @@
+prometheus:
```

Comment:
Would recommend to put this docker-compose.yml to `metrics` folder as well, so that indexer can only download this folder to their machine in one time.

Need to update the path in this file as well:

```
./metrics/prometheus.yml:/etc/prometheus/prometheus.yml -> ./prometheus.yml:/etc/prometheus/prometheus.yml

etc

```